Common/Data/Custom/CBOE/CBOE.cs

```diff
@@ -73,27 +86,27 @@ public override BaseData Reader(SubscriptionDataConfig config, string line, Date
             decimal low;
             decimal close;
 
-            if (!decimal.TryParse(csv[1], NumberStyles.Any, CultureInfo.InvariantCulture, out open))
+            if (!QuantConnect.Parse.TryParse(csv[1], NumberStyles.Any, out open))
```

Comment:
Per discussion has valid values for close so shouldn't reject the whole row/point.